src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -54,16 +73,42 @@ export default {
 		};
 	},
 	async mounted() {
-		this.allFacets = await fetchLoanFacets(this.apollo);
+		this.allFacets = await fetchLoanFacets(this.apollo, true);
```

Comment:
What is the `true` here? I'm only seeing one param for `fetchLoanFacets`.

Reply:
oops, removing this. was testing something out